app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -37,6 +42,24 @@
   font-size: medium;
 }
 
+.addPartnerDisplay {
+  background: white
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Declaration should be terminated by a semicolon